index.es.js

```diff
@@ -1,12 +1,5 @@
-export {
-  Future,
-  Future as default,
-  isFuture,
-  reject,
-  of,
-  never,
-  isNever
-} from './src/core';
+export {default, default as Future} from './src/future';
```

Comment:
Not 100% sure that ES specification allows to do `export {default} from '...'`. It maybe the case that it works with current tools by incident, and may not work in the future.